### PR TITLE
See commits for notes

### PR DESCRIPTION
**Pull request by derek**

### CODE REVIEW

**Comment by rgrove** [comment]
Thanks Derek! The YUI-related changes look good to me.

@paulirish, let me know when you've had a chance to review the jQuery changes and I'll get this sucker merged.

File: index.html [hunk 1]
```diff
@@ -393,9 +405,20 @@ Y.Node.create(&#x27;&lt;p&gt;foo&lt;/p&gt;&#x27;)
...
+      <td class="notes">Make DOM nodes appear/disappear</td>
+    </tr>
 
-<i>// If the transition module is loaded, a fade transition
-// will be used when true is passed.</i>
+    <tr>
+      <td class="code">
+<pre class="code jq">
+.fadeIn()
+.faceOut()
```
Comment (paulirish) [inline index.html:417]: smileyFaceOut()  :)

**Comment by paulirish** [comment]
Aside from that small typo, the rest look good.

File: index.html (hunk 1 above)
Comment (rgrove) [inline index.html:417]: Should add faceOff() for full Travolta power.

**Pull request merged**